testdata/simple2/docs/docs.go

```diff
@@ -1,17 +1,19 @@
 // GENERATED BY THE COMMAND ABOVE; DO NOT EDIT
 // This file was generated by swaggo/swag at
-// 2019-06-25 15:43:55.76642234 +0200 CEST m=+2.220403193
+// 2019-07-15 21:17:57.384551624 +0900 JST m=+0.027691337
 
 package docs
 
 import (
 	"bytes"
+	"encoding/json"
```

Comment:
Diffs are appeared after running `make test` (Probably not related to changes by this PR...) Please let me know if there are some problems :pray:

Reply:
It's ok for it.